src/scenedit/scen.sdfpicker.hpp

```diff
@@ -16,6 +16,8 @@ class cTilemap;
 
 class cStuffDonePicker {
 	cDialog dlog;
+	// Note: x and y in initial_sdf and chosen_sdf are actually (c, r) and must be flipped when
```

Comment:
There's no harm in this comment. I think it's not really necessary, because "x is column and y is row" is just a simple mathematical fact. But I guess maybe it's not as obvious to some people.